.changeset/shaggy-hairs-peel.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gradio/dropdown": minor
+"gradio": minor
+---
+
+fix:Add `.type` event listener to `gr.Dropdown()`
```

Comment:
```suggestion
fix:Add `.key_down` event listener to `gr.Dropdown()`
```

Reply:
Weird, maybe b/c this wasn't initially targeting `main`, now it is